pkg/pgmodel/old_migrator.go

```diff
@@ -0,0 +1,438 @@
+// This file and its contents are licensed under the Apache License 2.0.
```

Comment:
is this the old system verbatim or were there any logic changes. Hard to tell from this diff